cmd/api/src/database/migration/migrations/schema.sql

```diff
@@ -123,8 +123,8 @@ CREATE SEQUENCE IF NOT EXISTS asset_group_selectors_id_seq
 ALTER SEQUENCE asset_group_selectors_id_seq OWNED BY asset_group_selectors.id;
 
 CREATE TABLE IF NOT EXISTS asset_groups (
-    name text,
-    tag text,
+    name text UNIQUE NOT NULL,
+    tag text UNIQUE NOT NULL,
```

Comment:
It's my understanding that this schema.sql is not run a second time and is more akin to "migration 0" so changes to this file won't be reflected and we should use another migration .sql file. But please get confirmation on that one! 🙇

Reply:
Removed changes in `schema.sql` and applied to `v5.12.0.sql`